### ISSUE TITLE
3D render optimization reasearch

### ISSUE THREAD

**Issue by HarelM** [IsraelHikingMap/VectorMap#135]
It's worth looking into the following optimization of the style when it comes to 3D.
See the following comment:
https://github.com/maplibre/maplibre-gl-js/issues/1368#issuecomment-1232547588
While I'm not suggesting changing the style just yet, considering this improvement might improve the current rendering time (maybe) and worth trying out to see if we can create a style that will allow using the 3D feature (maybe as a new layer and not part of the current layers... just a though, we can discuss this more).
Bottom line, It would be great if we can create a style that doesn't crash our app when using the 3D feature (not to blame anyone for this obviously) :-)

**Comment by zstadler** [comment]
ilMTB style is not optimized yet
